src/examples/only-install.yml

```diff
@@ -0,0 +1,22 @@
+description: >
+  Alternatively, you can use the Snyk orb to only install the Snyk CLI, and then run the Snyk CLI commands in your own steps.
+usage:
+  version: 2.1
+
+  orbs:
+    snyk: snyk/snyk@x.y.z
+
+  jobs:
+    build:
+      docker:
+        - image: cimg/node:lts
+      steps:
+        - checkout
+        - run: npm ci
```

Comment:
Why this? this may not even be an npm project...

Reply:
I replicated one example written in `quickstart-app-scanning.yml` to show how to do that in multiple commands.